addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -242,44 +320,32 @@ export class AnalyticDistribution extends Component {
 
     // Editing Distributions
     async onSelect(option, params, tag) {
```

Comment:
What's the difference between this `onSelect` and `onSelected` in the `SelectCreateDialog`?

Reply:
onSelect > the function called when an autocomplete option is selected
onSelected > the function called when the select button in the SelectCreateDialog is clicked (providing the ids of the selected accounts)